doc/ap_verify_testdata/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _ap_verify_testdata-package:
+
+##################
+ap_verify_testdata
+##################
+
+The ``ap_verify_testdata`` package contains several files from `obs_test <https://github.com/lsst/obs_test/>`_.
+The package is intended to test dataset support in ``ap_verify`` code.
+
+Project info
+============
+
+Repository
+   https://github.com/lsst-dm/ap_verify_testdata
+
+.. Datasets do not have their own (or a collective) Jira components; by convention we include them in ap_verify
+
+Jira component
+   `ap_verify <https://jira.lsstcorp.org/issues/?jql=project %3D DM %20AND%20 component %3D ap_verify %20AND%20 text ~ "testdata">`_
+
+Dataset Contents
+================
+
+This package provides a number of demonstration files copied from `obs_test <https://github.com/lsst/obs_test/>`_.
+See that package for detailed file and provenance information.
+
+The dataset contents include raw images, biases, flats, and defects.
+The dataset does not provide any templates.
+It does have a small Gaia DR1 reference catalog, but it is not guaranteed to overlap with the footprint of the raw data.
```

Comment:
If the refcat doesn't overlap with the raw data... what is the point? (I'm not overly worried about this. It's just weird.) As in the other PR, please elaborate something like templates --> "templates for difference imaging"

Reply:
It lets us test refcat handling and "ingestion".

I'd love to have a more relevant refcat, but I don't know how to do that. For now this is just the same file that lived in `ap_verify_dataset_template`.